llm_toolkit/models.py

```diff
@@ -350,6 +350,16 @@ class GeminiPro(GeminiModel):
   _vertex_ai_model = 'gemini-1.0-pro'
 
 
+class GeminiExperimental(VertexAIModel):
+  """Gemini Experimental, results maybe genius or delightfully unpredictable."""
```

Comment:
Let's remove the "maybe genius or delightfully unpredictable." part :)  I know this is what is shown on Vertex AI, but no need to include it here in our codebase.

Reply:
Done